pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -6572,12 +6610,18 @@ func TestReconcilePipeline_FinalTasks(t *testing.T) {
 					t.Errorf("Expected PipelineRun status to be failed, but was %v for %s",
 						reconciledRun.Status.GetCondition(apis.ConditionSucceeded), tt.name)
 				}
+				if d := cmp.Diff(reconciledRun.Status.TaskRuns, tt.expectedTaskRuns); d != "" {
```

Comment:
This was added under the `pipelineRunStatusFalse` check as it has not reached https://github.com/tektoncd/pipeline/blob/main/pkg/reconciler/pipelinerun/pipelinerun.go#L230 yet so we have to check on the `status.taskrun`